payments/migrations/0028_update_order_statuses.py

```diff
@@ -0,0 +1,87 @@
+    OrderLogEntry = apps.get_model("payments", "OrderLogEntry")
+
+    Order.objects.filter(status="waiting").update(status="offered")
+    OrderLogEntry.objects.filter(from_status="waiting").update(from_status="offered")
+    OrderLogEntry.objects.filter(to_status="waiting").update(to_status="offered")
+
+
+def revert_order_statuses(apps, schema_editor):
+    Order = apps.get_model("payments", "Order")
+    OrderLogEntry = apps.get_model("payments", "OrderLogEntry")
+
+    Order.objects.filter(status="offered").update(status="waiting")
+    OrderLogEntry.objects.filter(from_status="offered").update(from_status="waiting")
+    OrderLogEntry.objects.filter(to_status="offered").update(to_status="waiting")
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("payments", "0027_berthswitchoffer_offer_number"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="order",
+            name="status",
+            field=models.CharField(
+                choices=[
+                    ("drafted", "Drafted"),
+                    ("offered", "Offered"),
+                    ("rejected", "Rejected"),
+                    ("cancelled", "Cancelled"),
+                    ("expired", "Expired"),
+                    ("error", "Error"),
+                    ("paid", "Paid"),
+                    ("paid_man", "Paid manually"),
+                    ("refunded", "Refunded"),
+                ],
+                default="drafted",
+                max_length=9,
+            ),
+        ),
+        migrations.AlterField(
+            model_name="orderlogentry",
+            name="from_status",
+            field=models.CharField(
+                choices=[
+                    ("drafted", "Drafted"),
+                    ("offered", "Offered"),
+                    ("rejected", "Rejected"),
+                    ("cancelled", "Cancelled"),
+                    ("expired", "Expired"),
+                    ("error", "Error"),
+                    ("paid", "Paid"),
+                    ("paid_man", "Paid manually"),
+                    ("refunded", "Refunded"),
+                ],
+                max_length=9,
+            ),
+        ),
+        migrations.AlterField(
+            model_name="orderlogentry",
+            name="to_status",
+            field=models.CharField(
+                choices=[
+                    ("drafted", "Drafted"),
+                    ("offered", "Offered"),
+                    ("rejected", "Rejected"),
+                    ("cancelled", "Cancelled"),
+                    ("expired", "Expired"),
+                    ("error", "Error"),
+                    ("paid", "Paid"),
+                    ("paid_man", "Paid manually"),
+                    ("refunded", "Refunded"),
+                ],
+                max_length=9,
+            ),
+        ),
+        migrations.RunPython(change_order_statuses, reverse_code=revert_order_statuses),
```

Comment:
Does this work once you've removed `waiting` from the options?

Reply:
Tested locally and this works both ways. I think it works because of the `update()` instead of `save()`.

Reply:
So I guess the database does not check and enforce that you only have one of the choices, so I guess it would work?

Reply:
Yes, exactly.